### ISSUE TITLE
FR: Apply colors to rendered LaTeX expressions

### ISSUE THREAD

**Issue by turtlebasket** [zsviczian/obsidian-excalidraw-plugin#1084]
**Is your feature request related to a problem? Please describe.**

Not related to a problem, mostly just an issue of consistency functionality that would be nice to have.

**Describe the solution you'd like**

Coloring options for LaTeX expressions, much like with the native text functionality.

**Describe alternatives you've considered**

N/A

**Additional context**

<img width="537" alt="Screen Shot 2023-04-07 at 3 12 05 PM" src="https://user-images.githubusercontent.com/32886427/230685217-e2fc7d64-e222-446b-9a41-5bc45be038b2.png">

Since I'm unfamiliar with the guts of Excalidraw, I'm not too sure how achievable this is, maybe there's a way to trick it into thinking the rendered SVG is a re-colorable line or something?

**Comment by zsviczian** [comment]
recoloring SVG is certainly on my list of things I want to develop.
For LaTeX, however, there is an easier solution.
Use the insert LaTeX formula command from the Tools Menu and type `\color{lightcoral}\binom{n}{k} = \frac{n!}{k!(n-k)!}`
of course only the `\color{#FF0000}` part is important. You can use hexadecimal colors, or [color names](https://www.w3schools.com/colors/colors_names.asp), and probably other formats as well, but I haven't tried that.

![image](https://user-images.githubusercontent.com/14358394/230756470-7bc62e52-89d7-43d8-97d4-0cc964269cc3.png)

See also: https://youtu.be/r08wk-58DPk

**Comment by turtlebasket** [comment]
I agree that this does work. But I think broadly speaking, it would "make more sense" to be able to select multiple things (arrows, boxes, LaTeX exprs, etc) and recolor them all at once from the built-in Excalidraw palette.

**Comment by zsviczian** [comment]
It would make sense. The issue is, I don't use LaTeX, ever. As such my level of energy to put in the day of work required to make this work is just not there. I am sorry.
